docs/using/recipe_code/custom_environment.dockerfile

```diff
@@ -28,12 +28,16 @@ RUN "${CONDA_DIR}/envs/${env_name}/bin/python" -m ipykernel install --user --nam
 RUN "${CONDA_DIR}/envs/${env_name}/bin/pip" install --no-cache-dir \
     'flake8'
 
-# Uncomment this section to activate custom environment by default
+# This changes the custom Python kernel so that the custom environment will
+# be activated for the respective Jupyter Notebook and Jupyter Console
+RUN sed -i.bak ":a;N;\$!ba;s| }\n}| },\n \"env\": {\n  \"XML_CATALOG_FILES\": \"\",\n  \"PATH\": \"${CONDA_DIR}/envs/${env_name}/bin:\$PATH\",\n  \"CONDA_PREFIX\": \"${CONDA_DIR}/envs/${env_name}\",\n  \"CONDA_PROMPT_MODIFIER\": \"\(${env_name}\) \",\n  \"CONDA_SHLVL\": \"2\",\n  \"CONDA_DEFAULT_ENV\": \"${env_name}\",\n  \"CONDA_PREFIX_1\": \"${CONDA_DIR}\"\n }\n}|g" "/home/${NB_USER}/.local/share/jupyter/kernels/${env_name}/kernel.json"
```

Comment:
May I ask you to rewrite this in Python?
It's so difficult to read and understand.
You can write inline multiline Python using [heredoc](https://stackoverflow.com/a/58833680/4881441)

Reply:
I am no Python guy...

```bash
diff /home/${NB_USER}/.local/share/jupyter/kernels/python310/kernel.json.bak /home/${NB_USER}/.local/share/jupyter/kernels/python310/kernel.json
```

```diff
12a13,21
>  },
>  "env": {
>   "XML_CATALOG_FILES": "",
>   "PATH": "/opt/conda/envs/python310/bin:$PATH",
>   "CONDA_PREFIX": "/opt/conda/envs/python310",
>   "CONDA_PROMPT_MODIFIER": "(python310) ",
>   "CONDA_SHLVL": "2",
>   "CONDA_DEFAULT_ENV": "python310",
>   "CONDA_PREFIX_1": "/opt/conda"
```

Using `${CONDA_DIR}` instead `/opt/conda` and `${env_name}` instead of `python310` with the `sed` command.